tests/examples/test_contrib/test_sqlalchemy/plugins/test_tutorial_example_apps.py

```diff
@@ -12,42 +13,34 @@
 )
 from sqlalchemy.ext.asyncio import create_async_engine
 
-from litestar import Litestar
 from litestar.testing import TestClient
 
 
-@pytest.fixture(
-    params=[
+@pytest.mark.parametrize(
+    "app_module",
+    [
         full_app_no_plugins,
         full_app_with_init_plugin,
         full_app_with_plugin,
         full_app_with_serialization_plugin,
         full_app_with_session_di,
-    ]
+    ],
 )
-async def app(monkeypatch: MonkeyPatch, request: FixtureRequest) -> Litestar:
```

Comment:
The point of this being a fixture was to ensure we get a new engine per app per module, so the tests are properly isolated. Please revert

Reply:
done